src/pages/main/hooks/useTemporaryFileNodes.ts

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useState } from "react";
+
+import { TNodeUid } from "@_node/types";
+
+export const useTemporaryFileNodes = () => {
```

Comment:
Let's add some comments to explain what they do! It's not very obvious why and when to use this hook.

Reply:
sure

Reply:
at the File System level, when `rename` directory/file
- it is copied first and after copying is done, it is renamed and deletes the original one.

before, file-tree sync was done in real time, so at that time we could see one more tmp folder while `renaming`. so I used that.
but following the current file system sync logic, we will not need that I guess.